### PR TITLE
HTML Reporter: Escape setUrl output

### PR DESCRIPTION
**Pull request by leobalter**

### CODE REVIEW

**Comment by leobalter** [comment]
merged at  a5806f1

File: reporter/html.js [hunk 1]
```diff
@@ -422,7 +422,7 @@ function appendHeader() {
 
 	if ( header ) {
 		header.innerHTML = "<a href='" +
-			setUrl({ filter: undefined, module: undefined, testId: undefined }) +
+			escapeText( setUrl( { filter: undefined, module: undefined, testId: undefined } ) ) +
```
Comment (trentmwillis) [inline reporter/html.js]: Why not just update `setUrl` to do the escaping?

Comment (gibson042) [inline reporter/html.js]: `setUrl` output doesn't always get written into HTML: https://github.com/jquery/qunit/blob/a5806f1178b02b9655f19569ac21adf510d8c62f/reporter/html.js#L260-L274

**Comment by leobalter** [comment]
I forgot to close this, it's merged

**Pull request closed without merging**